lib/Jsrt/Jsrt.cpp

```diff
@@ -1244,7 +1244,7 @@ CHAKRA_API JsGetGlobalObject(_Out_ JsValueRef *globalObject)
 
 CHAKRA_API JsCreateObject(_Out_ JsValueRef *object)
 {
-    return ContextAPINoScriptWrapper([&](Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
+    return ContextAPIWrapper<true>([&](Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
```

Comment:
Why do these calls need to change from `ContextAPINoScriptWrapper` to `ContextAPIWrapper`? Is this fixing a correctness bug? It looks to me like creating an object doesn't rely on executing any script.